README.md

```diff
@@ -7,68 +7,131 @@ PADD (formerly Chronometer2) is a more expansive version of the original chronom
 ***Note:** PADD has been adopted by the Pi-hole team, thanks to JPMCK for creating this helpful tool!
 
 ## Setup PADD
-*More in-depth information about setting up PADD can be found in this repo’s [wiki](https://github.com/pi-hole/PADD/wiki/Setup).*
 
 - Get a copy of PADD by running:
-
-```bash
-cd ~
-wget -O padd.sh https://install.padd.sh
-```
-or
-```bash
-cd ~
-curl -sSL https://install.padd.sh -o padd.sh
-```
+  ```bash
+  cd ~
+  wget -O padd.sh https://install.padd.sh
+  ```
+  or
+  ```bash
+  cd ~
+  curl -sSL https://install.padd.sh -o padd.sh
+  ```
 
 - Make PADD executable by running
-
-```bash
-sudo chmod +x padd.sh
-```
-
-- Set PADD to auto run by adding the following to the end of `~/.bashrc`:
-
-```bash
-# Run PADD
-# If we’re on the PiTFT screen (ssh is xterm)
-if [ "$TERM" == "linux" ] ; then
-  while :
-  do
-    ./padd.sh
-    sleep 1
-  done
-fi
-```
-
-One line version
-
-```bash
-cd ~ ; echo "if [ \"\$TERM\" == \"linux\" ] ; then\n  while :\n  do\n    ./padd.sh\n    sleep 1\n  done\nfi" | tee ~/.bashrc -a
-```
-
-- Reboot your Pi-Hole by running `sudo reboot`. PADD should now run when your Pi-Hole has completed booting.
+  ```bash
+  sudo chmod +x padd.sh
+  ```
+
+## Using PADD
+
+### PADD on Pi-hole machine
+
+- if running as user belonging to the group `pihole`, if not `--secret` wii be needed
```

Comment:
```suggestion
- if running as user belonging to the group `pihole`, if not `--secret` will be needed
```

> if not `--secret` will be needed

This is not true, this is optional. If not provided, we ask for the password during startup

Maybe it's best just to write something short as  "Just run"